presto-spi/src/main/java/com/facebook/presto/spi/block/ClosingBlockLease.java

```diff
@@ -65,6 +65,7 @@ public void close()
             return;
         }
         closed = true;
+        block = null;
```

Comment:
This change is problematic. The lease doesn't own the block and the same block can be used to create multiple leases.

Reply:
All the usage seems to create a new block everytime before getting the lease

Reply:
@bhhari Actually, I got confused. This is fine as lease can not be used after "close()".